content/faq.md

```diff
@@ -14,9 +14,25 @@ Dunst was unable to connect to the X server. Make sure that X is running and tha
 
 If you're using the systemd service, that might mean that dbus isn't setting the right variables, see issue [#347](https://github.com/dunst-project/dunst/issues/347) for more details.
 
+## Cannot connect to DBus
+
+Dunst cannot connect to you session bus at all. The session bus might not even exist.
+
+Please check all these conditions:
+
+- dbus packages are really installed
+- dbus is running
+- The env variable `DBUS_SESSION_BUS_ADDRESS` is really set
+
+## Cannot acquire 'org.freedesktop.Notifications'
+
+Another notification daemon is already running, which is listening for notifications. Usually this is another daemon, which got autostarted or is dunst itself.
+
+Usually, dunst also prints the PID of this process, which will give you more possibilities to investigate. Maybe you want to simply `kill` this PID and then start dunst.
+
 ## Name Lost. Is Another notification daemon running?
```

Comment:
I don't see a reason in keeping this, the `Name Lost [...]` error was removed in favour of `Cannot connect to DBus` and `Cannot acquire [...]`.

Reply:
Well, there are lots of others still using 1.2 and previous releases. So a proper description doesn't hurt on the website.

Reply:
Good point!